packages/flutter_tools/lib/src/resident_runner.dart

```diff
@@ -458,6 +459,38 @@ class FlutterDevice {
     logReader.appPid = vm.pid;
   }
 
+  Future<void> _startDartDevelopmentService(
```

Comment:
In the case of multiple devices, so we need to start up multiple dds instances?

Reply:
Assuming that it's currently possible to debug multiple devices at once, each instance of the VM service needs a matching DDS instance.